src/Sql/dbo/Tables/TaxRate.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE [dbo].[TaxRate] (
+    [Id]                VARCHAR(40)         NOT NULL        PRIMARY KEY,
+    [Country]           VARCHAR(50)         NOT NULL,
+    [State]             VARCHAR(2)          NULL,
+    [PostalCode]        VARCHAR(10)         NOT NULL,
+    [Rate]              DECIMAL(5,2)        NOT NULL,
+    [Active]        BIT                 NOT NULL
+);
+GO
+
+CREATE UNIQUE INDEX IX_TaxRate_Country_PostalCode_Active_Uniqueness
```

Comment:
Proper script execution for updates on some SQL systems require square brackets around names, please add square brackets around the index name.